### ISSUE TITLE
On cropSize change prop doesn't trigged computeSize or updating state of state.cropSize

### ISSUE THREAD

**Issue by Lopliok** [ValentinH/react-easy-crop#467]
**Describe the bug**
When property cropSize is changed in same time as zoom, computeSizes is not called. So state of cropSize is not updated.

**Expected behavior**
If property cropSize has changed, it will update Resizer state cropSize

![image](https://github.com/ValentinH/react-easy-crop/assets/4508201/00567a4b-bf28-4d2a-bb19-a4667a14c2aa)

We are using onMediaLoaded callback to calculate zoom and cropSize. Also we use range sliders to manipulate width, height, zoom.

I tried move condition in componentDidUpdate to be separately and called everytime when cropSize changed and it helps. Should I create PR for this?

![image](https://github.com/ValentinH/react-easy-crop/assets/4508201/14aecead-4c39-4ad2-bef2-12d1d8dfe088)

**Comment by ValentinH** [comment]
Indeed, having all the conditions tied together doesn't seem right. Most of else should be removed I think.

By the way, why are you using `cropSize` instead of `aspect`?

**Comment by Lopliok** [comment]
I made some local patch for our case, but if i will have time i'll will try investigate how can be helpful aspect.
